shaolinmkz/iReporter
# #162178946 build endpoint to create a red-flag record

#### What does this PR do?
Have the create incident record API functional

#### Description of Task to be completed?
Having the endpoint POST  `host:port/api/v1/red-flags` working

#### How should this be manually tested?
After cloning the repo, cd into it then:
* RUN `npm test` to see test result
* RUN `npm start` to start the server
* Using Postman test the endpoint POST  `host:port/api/v1/red-flags` with the header:
_key:_ Content-Type value: application/json

#### What are the relevant pivotal tracker stories?
[#162178946](https://www.pivotaltracker.com/story/show/162178946)
